colabseg/membrainseg_wrapper.py

```diff
@@ -0,0 +1,93 @@
+#
+# ColabSeg - Interactive segmentation GUI
+#
+# Marc Siggel, December 2021
+# Adapted by Lorenz Lamm for MemBrain-seg, July 2023
+
+import os
+from ipywidgets import widgets
+from ipywidgets import Layout
+
+from membrain_seg.segmentation.segment import segment as membrain_segment
+
+
+def generate_membrainseg_gui():
+    """Generates MemBrain-seg GUI"""
+    all_values = {}
+
+    def run_membrain_code(obj):
+        """This is a wrapper for MemBrain-seg"""
+        print("MemBrain-seg running on {}".format(all_values["tomo_file"].value))
+        if os.path.isfile(all_values["tomo_file"].value) is False:
+            raise ValueError("Tomogram doesn't exist!")
+
+        print("Segmenting your tomogram using MemBrain-seg.")
+        out_file_seg = membrain_segment(
+            tomogram_path=all_values["tomo_file"].value,
+            ckpt_path=all_values["membrain_model"].value,
+            out_folder=os.getcwd(),
+            store_connected_components=all_values["compute_connected_components"].value,
+            connected_component_thres=(None if all_values["connected_component_thres"].value == "" else float(all_values["connected_component_thres"].value)),
+            test_time_augmentation=all_values["test_time_augmentation"].value
+        )
+
+        print("Your file is written to", out_file_seg)
```

Comment:
Maybe we should have an option to specify a target path where the segmentation should be stored?